tox.ini

```diff
@@ -16,6 +16,8 @@ commands =
 deps = -rrequirements/test-cpu.txt
 commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
+    python -m pip install .
```

Comment:
Turns out installing the version-under-test is important, since Triton uses the version that's installed in the tests.

Reply:
we don't currently test with Triton in the GitHub actions tests. So I assume this comment was intended to refer to the change below on the `test-gpu` env. No harm running the install here too, but I suppose not strictly required.

I recall that this wasn't possible before due to the way we were running the tests in Jenkins. To check my understanding: was this possibility of making Triton pick up the version-under-test unlocked by the transition to tox because it's now running in a virtual env that is isolated from the other tests being run?

Reply:
I think you have it exactly right. The installed version shouldn't matter in CPU, but on GPU using tox lets us install the version in the current PR without stomping on the version installed in the CI container that's used by the tests in other repos (currently just `Merlin`.)